src/agent/src/config.rs

```diff
@@ -1020,6 +1035,17 @@ mod tests {
                 guest_components_procs: GuestComponentsProcs::None,
                 ..Default::default()
             },
+            #[cfg(feature = "guest-pull")]
+            TestData {
+                contents: "agent.image_registry_auth=file:///root/.docker/config.json",
+                image_registry_auth: "file:///root/.docker/config.json",
+                ..Default::default()
+            },
+            TestData {
```

Comment:
Is `#[cfg(feature = "guest-pull")]` needed here, too?

Reply:
Yeah, I think you are correct